routing_model/index.html

```diff
@@ -169,7 +169,7 @@ <h2>Becoming a full member of a section</h2>
     StartComputeResourceProofForElder["start_compute_resource_proof(source elder)<br/><br/>schedule(TimeoutRefused)"]
     StartComputeResourceProofForElder --> LoopEnd
 
-    Rpc -- "ExpectCandidate<br/>RefuseCandidate<br/>RelocateResponse<br/>..." --> VoteParsecRPC
+    Rpc -- "ExpectCandidate<br/>ExpectCandidateRefuseResponse<br/>ExpectCandidateAcceptResponse<br/>..." --> VoteParsecRPC
```

Comment:
I understand how the words: ExpectCandidateAcceptResponse and ExpectCandidateRefuseResponse came to be, but I find them a bit hard to parse. I think CandidateAccepted or CandidateRefused would read better, despite conveying a bit less explicitely that they are a response to the ExpectCandidate RPC.

In general, I would aim for short names that express just the amount of context needed to make sense of them, same as in code...

Reply:
Our RPCs in the code do have that format Name/NameResponse, If we want a shorter name, maybe we should look at the ExpectCandidate one. Could be `Relocating`, maybe.

Reply:
Uhm. What about this:
`Rpc::ExpectCandidate`,
`Rpc::ExpectCandidateResponse`,
But then, like `ExpectCandidate` contains a `CandidateInfo`, `ExpectCandidateResponse` could contain a `Result<RelocationInfo>`, where `RelocationInfo` would be a `struct` containing the target range and the destination section info.

Reply:
If the `result.is_err()`, of course it would mean a refusal. If that's not clear enough, we could have our own `enum RelocationResponse { Accepted, Refused };`

Reply:
That was my original proposal, but when we discussed it we did not seem to like the nesting, I kind of come around to it.

Reply:
I think it's OK with a caveat: we don't have to show the nesting on the graphs ;)